apps/ledger-live-mobile/src/navigation/DeeplinksProvider.tsx

```diff
@@ -71,7 +74,7 @@ function getProxyURL(url: string) {
   // This is to handle links set in the useFromAmountStatusMessage in LLC.
   // Also handles a difference in paths between LLD on LLD /platform/:app_id
   // but on LLM /discover/:app_id
-  if (hostname === "platform" && [DEFAULT_MULTIBUY_APP_ID].includes(platform)) {
+  if (hostname === "platform" && [DEFAULT_MULTIBUY_APP_ID, BUY_SELL_UI_APP_ID].includes(platform)) {
```

Comment:
Here do we want to add the `buySellUiFlag.params.manifestId` in the array ?

Reply:
Yes for completion, good plan, added